databricks/koalas/frame.py

```diff
@@ -10049,6 +10049,90 @@ def explode(self, column):
         internal = self._internal.copy(spark_frame=sdf, data_spark_columns=data_scols)
         return DataFrame(internal)
 
+    def mad(self, axis=0):
+        """
+        Return the mean absolute deviation of values.
+
+        Parameters
+        ----------
+        axis : {index (0), columns (1)}
+            Axis for the function to be applied on.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'a': [1, 2, 3, np.nan], 'b': [0.1, 0.2, 0.3, np.nan]},
+        ...                   columns=['a', 'b'])
+
+        >>> df.mad()
+        a    0.666667
+        b    0.066667
+        Name: 0, dtype: float64
+
+        >>> df.mad(axis=1)
+        0    0.45
+        1    0.90
+        2    1.35
+        3     NaN
+        Name: 0, dtype: float64
+        """
+        from databricks.koalas import Series
+        from databricks.koalas.series import first_series
+
+        axis = validate_axis(axis)
+
+        if axis == 0:
+            exprs = []
+            new_column_labels = []
+            for label in self._internal.column_labels:
+                col_sdf = self._internal.spark_column_for(label)
```

Comment:
I would name it like `scol` or `new_scol`

Reply:
I Agree. 
Renamed it to scol.